web/index.html

```diff
@@ -16,5 +17,6 @@
     }}();
     </script>
     <script type="module" src="/src/main.tsx"></script>
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
do we need this even after installing the `react-github-button` package?

Reply:
Github button package will be removed 

I have to do the cleanup i was testing

It doesn't work as expected and needed to have manual github button